README.md

```diff
@@ -26,3 +26,16 @@ Repository to describe, develop, document and test the Device Swap API family
     <!-- Note: the $sub-project-mailinglistname$ is either already existing or will be created by the CAMARA Admin Team  -->
     * Subscribe / Unsubscribe to the mailing list of this Sub Project <https://lists.camaraproject.org/g/sp-dsw>.
     * A message to the community of this Sub Project can be sent using <sp-dsw@lists.camaraproject.org>.
+
+## Status and released versions
```

Comment:
This sections between "Scope" and "Contributing", title in the template is "Release Information", cf. https://github.com/camaraproject/Template_API_Repository/blob/main/README.md

Reply:
Moved and title updated